constants/status_codes.go

```diff
@@ -0,0 +1,71 @@
+package constants
+
+var StatusTexts = map[int]string{
+	100: "Continue",
+	101: "Switching Protocols",
+	102: "Processing",
+	103: "Early Hints",
+
+	200: "OK",
+	201: "Created",
+	202: "Accepted",
+	203: "Non-Authoritative Information",
+	204: "No Content",
+	205: "Reset Content",
+	206: "Partial Content",
+	207: "Multi-Status",
+	208: "Already Reported",
+	226: "IMUsed",
```

Comment:
This map should be added in `constants/constants.go` i don't think there's need to create a new file.
and i doubt if we gonna use soo many values so just limit it to the below mentioned one
```go
const HTTPStatusCodes := map[int]string{
	http.StatusOK:                   http.StatusText(http.StatusOK),
	http.StatusCreated:              http.StatusText(http.StatusCreated),
	http.StatusAccepted:             http.StatusText(http.StatusAccepted),
	http.StatusNonAuthoritativeInfo: http.StatusText(http.StatusNonAuthoritativeInfo),
	http.StatusNoContent:            http.StatusText(http.StatusNoContent),
	http.StatusBadRequest:           http.StatusText(http.StatusBadRequest),
	http.StatusUnauthorized:         http.StatusText(http.StatusUnauthorized),
	http.StatusForbidden:            http.StatusText(http.StatusForbidden),
	http.StatusNotFound:             http.StatusText(http.StatusNotFound),
	http.StatusMethodNotAllowed:     http.StatusText(http.StatusMethodNotAllowed),
	http.StatusConflict:             http.StatusText(http.StatusConflict),
	http.StatusInternalServerError:  http.StatusText(http.StatusInternalServerError),
}
```